data/overview.yaml

```diff
@@ -49,12 +49,12 @@ linear algebra:
   kernel of a linear map: 'linear_algebra/basic.html#linear_map.ker'
   quotient space: 'linear_algebra/basic.html#submodule.quotient'
   sum of subspaces: 'linear_algebra/basic.html#submodule.complete_lattice'
-  direct sum: ''
+  direct sum: 'data/finsupp.html#finsupp.vector_space'
```

Comment:
This is direct sums... but not dependent. And `dfinsupp` seems to be gone. Alternatively, we could link to the vector space structure on `V × W`.

Reply:
What is intended here is clearly the bigop $\bigoplus V_i$.